From c439d65b38ad15cd6bcf9a2c135afe82427a1ea9 Mon Sep 17 00:00:00 2001
From: Gyorgy Sarvari <skandigraun@gmail.com>
Date: Wed, 15 May 2024 20:20:45 +0200
Subject: [PATCH] Don't use virtual env for building

Yocto builds are running in a sandbox by default, it is not needed
to run the build inside a python virtual environment. Beside that
there is no network connection during build, so setting up the
environment is also problematic.
---
 python/mach_bootstrap.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/python/mach_bootstrap.py b/python/mach_bootstrap.py
index 6690bc1bd2c..433374471d1 100644
--- a/python/mach_bootstrap.py
+++ b/python/mach_bootstrap.py
@@ -227,7 +227,7 @@ def bootstrap(topdir):
         print('You are running Python', platform.python_version())
         sys.exit(1)
 
-    _activate_virtualenv(topdir)
+#    _activate_virtualenv(topdir)
 
     def populate_context(context, key=None):
         if key is None:
-- 
2.45.0

